cls/SourceControl/Git/Settings.cls

```diff
@@ -110,12 +113,16 @@ Method %Save() As %Status
     set @storage@("settings", "compileOnImport") = ..compileOnImport
     set @storage@("settings", "warnInstanceWideUncommitted") = ..warnInstanceWideUncommitted
     set @storage@("settings", "basicMode") = ..systemBasicMode
+    set @storage@("settings", "environmentName") = ..environmentName
     if ..basicMode = "system" {
         kill @storage@("settings", "user", $username, "basicMode")
     } else {
         set @storage@("settings", "user", $username, "basicMode") = ..basicMode
     } 
 
+    set ^%SYS("SystemMode") = ..environmentName
```

Comment:
Nitpick: might prefer calling $SYSTEM.Version.SystemMode(environment name) instead of setting the global directly.